docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -141,44 +149,148 @@ Restrictions
-p. 112 of [GBD-2017-Risk-Appendix-VAD]_.
+p. 112 of [GBD-2017-Risk-Appendix-VAD]_, copied here for reference:
+
+.. _gbd_2017_vad_relative_risk_table:
 
 .. list-table:: Pooled relative risks for risk-outcome pairs included in GBD 2017
-	:widths: 15 13 15 15
-	:header-rows: 1
-
-	* - Cause
-	  - GBD 2016 RR
-	  - GBD 2017 RR
-	  - Include in GBD 2017
-	* - Diarrhea
-	  - 1.6 (1.21 - 2.02)
-	  - 2.35 (2.17 - 2.54)
-	  - Yes
-	* - Measles
-	  - 2.4 (1.61 - 3.48)
-	  - 2.76 (2.01 - 3.78)
-	  - Yes
-	* - Lower Respiratory Infections (LRI)
-	  - 
-	  - 1.23 (1.03 - 1.48)
-	  - Yes
-	* - Meningitis
-	  - 
-	  - 3.2 (0.69 - 14.75)
-	  - No (not significant)
-	* - Malaria
-	  - 
-	  - 3.65 (2.23 - 5.97)
-	  - No (only one study)
-	  
+  :widths: 15 13 15 15
+  :header-rows: 1
+
+  * - Cause
+    - GBD 2016 RR
+    - GBD 2017 RR
+    - Include in GBD 2017
+  * - Diarrhea
+    - 1.6 (1.21 - 2.02)
+    - **2.35 (2.17 - 2.54)**
+    - Yes
+  * - Measles
+    - 2.4 (1.61 - 3.48)
+    - **2.76 (2.01 - 3.78)**
+    - Yes
+  * - Lower Respiratory Infections (LRI)
+    -
+    - **1.23 (1.03 - 1.48)**
+    - Yes
+  * - Meningitis
+    -
+    - 3.2 (0.69 - 14.75)
+    - No (not significant)
+  * - Malaria
+    -
+    - 3.65 (2.23 - 5.97)
+    - No (only one study)
+
+The above relative risks for GBD 2017 should be interpreted as rate ratios for
+the incidence rates of diarrhea, measles, and LRI.
+
 Vivarium Modeling Strategy
 --------------------------
 
+We will use an **exposure model** (or **prevalence-only model** or **propensity
+model**) for a vitamin A deficiency, in which each simulant is initialized with a "propensity" for vitamin A deficiency, and the simulant's vitamin A status is determined by comparing this
+propensity to the overall VAD exposure/prevalence in the population.
+Such
+propensity/exposure models have been used in Vivarium for other risk factors and
+risk-attributable causes, such as child stunting, :ref:`child wasting/PEM
+<2017_cause_pem>`, and :ref:`iron deficiency anemia
+<2017_cause_iron_deficiency>`.
+
+In more detail, the basic strategy is to initialize each simulant with a
+propensity score distributed uniformly in [0,1], then compare this propensity
+score with the (location/age/sex/year/intervention-status)-dependent prevalence
+of vitamin A deficiency at each time step to determine whether the simulant has
+VAD during that time step. Each simulant's propensity is assigned only once, but
+the underlying prevalence distribution can change throughout the course of the
+simulation, which may result in a change in the simulant's vitamin A status.
+
+In particular, our modeling strategy will **not** explicitly use incidence or
+remission data for vitamin A deficiency, but only *prevalence* (which is the
+same as the exposure data for the VAD risk factor). The rationale for this approach is twofold:
+
+1.  We want to guarantee that the simulated baseline prevalence of vitamin A
```

Comment:
We are only doing this because of option 2.  We expect the cause version of the model to get incidence and remission correct in addition to getting prevalence correct.

Reply:
However, it is more important for us to use the intervention data correctly than it is to get the dynamic parameters of the disease correct.  Details about the limitations and the expected impact to be found in <assumptions and limitations section>.

Reply:
With regard to (1), there's also the concern mentioned in @aflaxman 's comment above: If we use the incidence and remission data from GBD, then basically the whole population will end up getting VAD if we run the simulation for 5 years, but we suspect this would not be representative of reality.

Reply:
I added a todo to clarify this.